distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Format.enso

```diff
@@ -64,9 +65,13 @@ type Excel_Format
        If the File_Format supports reading from the web response, return a configured instance.
     for_web : Text -> URI -> Excel_Format | Nothing
     for_web content_type uri =
-        _ = [content_type, uri]
+        _ = [uri]
         ## Currently not loading Excel files automatically as these need to be loaded as a connection.
```

Comment:
Is this comment still up to date???
We are loading excel files after all it seems?